code/__DEFINES/misc.dm

```diff
@@ -468,3 +468,6 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 #define ALIGNMENT_GOOD "good"
 #define ALIGNMENT_NEUT "neutral"
 #define ALIGNMENT_EVIL "evil"
+
+//Free gear from the loadout
+#define FREE_GEAR 0
```

Comment:
honestly now using the define is a bit silly (cause people will figure out what cost 0 means anyways so maybe remove that and just put 0 at those again

Reply:
before i recommended using a define because i tought you would try to do something different didn't think the issue was actually related to the check